pvlib/inverter.py

```diff
@@ -0,0 +1,305 @@
+    ac_power = np.where(p_dc < Pso, -1.0 * abs(Pnt), ac_power)
+
+    if isinstance(p_dc, pd.Series):
+        ac_power = pd.Series(ac_power, index=p_dc.index)
+
+    return ac_power
+
+
+def adr(v_dc, p_dc, inverter, vtol=0.10):
+    r'''
+    Converts DC power and voltage to AC power using Anton Driesse's
+    Grid-Connected PV Inverter efficiency model
+
+    Parameters
+    ----------
+    v_dc : numeric
+        DC voltage input to the inverter, should be >= 0. [V]
+
+    p_dc : numeric
+        DC power input to the inverter, should be >= 0. [W]
+
+    inverter : dict-like
+        Defines parameters for the inverter model in [1]_.  See Notes for
+        required model parameters. A parameter database is provided with pvlib
+        and may be read using :py:func:`pvlib.pvsystem.retrieve_sam.
+
+    vtol : numeric, default 0.1
+        Fraction of DC voltage that determines how far the efficiency model is
+        extrapolated beyond the inverter's normal input voltage operating
+        range. 0.0 <= vtol <= 1.0. [unitless]
+
+    Returns
+    -------
+    ac_power : numeric
+        AC power output. [W]
+
+    Notes
+    -----
+
+    Determines the AC power output of an inverter given the DC voltage and DC
+    power. Output AC power is bounded above by the parameter ``Pacmax``, to
+    represent inverter "clipping". AC power is bounded below by ``-Pnt``
+    (negative when power is consumed rather than produced) which represents
+    self-consumption. `ac_power` is not adjusted for maximum power point
+    tracking (MPPT) voltage windows or maximum current limits of the inverter.
+
+    Required model parameters are:
+
+    =======   ============================================================
+    Column    Description
+    =======   ============================================================
+    Pnom      Nominal DC power, typically the DC power needed to produce
+              maximum AC power output. [W]
+
+    Vnom      Nominal DC input voltage. Typically the level at which the
+              highest efficiency is achieved. [V]
+
+    Vmax      Maximum DC input voltage. [V]
+
+    Vmin      Minimum DC input voltage. [V]
+
+    Vdcmax    . [V]
+
+    MPPTHi    Maximum DC voltage for MPPT range. [V]
+
+    MPPTLow   Minimum DC voltage for MPPT range. [V]
+
+    Pacmax    Maximum AC output power, used to clip the output power
+              if needed. [W]
+
+    ADRCoefficients  A list of 9 coefficients that capture the influence
+              of input voltage and power on inverter losses, and thereby
+              efficiency. Corresponds to terms from [1]_ (in order): :math:
+              `b_{0,0}, b_{1,0}, b_{2,0}, b_{0,1}, b_{1,1}, b_{2,1}, b_{0,2},
+               b_{1,2},  b_{1,2}`.
+
+    Pnt       AC power consumed by inverter at night (night tare) to
+              maintain circuitry required to sense PV array voltage. [W]
+
+    =======   ============================================================
```

Comment:
Sphinx still doesn't like this.  I'm not if all of these steps are necessary, but I got it working locally for me by:

1) Adding spacing between terms and definitions so that there is no overlap between columns (the ADRCoefficients term sticks out pretty far)
2) Extending the left `====` strings so that they completely cover the column width
3) Removing the empty line between Pnt and the trailing line.

Here's the text that works for me (with some text reflows to stay under 80 chars):

```
    ===============   ========================================================
    Column            Description
    ===============   ========================================================
    Pnom              Nominal DC power, typically the DC power needed to
                      produce maximum AC power output. [W]
    Vnom              Nominal DC input voltage. Typically the level at which
                      the highest efficiency is achieved. [V]
    Vmax              Maximum DC input voltage. [V]
    Vmin              Minimum DC input voltage. [V]
    Vdcmax            . [V]
    MPPTHi            Maximum DC voltage for MPPT range. [V]
    MPPTLow           Minimum DC voltage for MPPT range. [V]
    Pacmax            Maximum AC output power, used to clip the output power
                      if needed. [W]
    ADRCoefficients   A list of 9 coefficients that capture the influence
                      of input voltage and power on inverter losses, and
                      thereby efficiency. Corresponds to terms from [1]_
                      (in order): :math:`b_{0,0}, b_{1,0}, b_{2,0}, b_{0,1},
                      b_{1,1}, b_{2,1}, b_{0,2}, b_{1,2}, b_{1,2}`. See [1]_
                      for the use of each coefficient and the associated unit.
    Pnt               AC power consumed by inverter at night (night tare) to
                      maintain circuitry required to sense PV array voltage.
                      [W]
    ===============   ========================================================
```

Reply:
Two more notes:  Vdcmax is still undefined and `b_{1,2}` is repeated in `ADRCoefficients`, presumably should be `b_{2,2}`?